docs/building-spark.md

```diff
@@ -273,6 +273,8 @@ If you are building Spark for use in a Python environment and you wish to pip in
 
 **Note:** Due to packaging requirements you can not directly pip install from the Python directory, rather you must first build the sdist package as described above.
 
+Alternatively, you can also run make-distribution with the --pip option.
```

Comment:
Do you need to document the `--r` option somewhere? 
If you make another pass, you might back-tick `--pip` for tidiness.